public/scss/components/Maintenance.module.scss

```
@@ -13,17 +13,42 @@
     {
       text-align: center;
       margin: 24px 0;
-
+      order: 1;
       &Img
       {
         width: 100px;
       }
     }
 
+    &Title
+    {
+      font-family: 'Inter', sans-serif !important;
+      font-style: normal !important;
+      font-weight: 400 !important;
+      font-size: 20px !important;
+      line-height: 24px !important;
+      margin-bottom: 8px !important;
+      max-width: 100% !important;
+      order: 2 !important;
```

```suggestion
      @include typographyBuilder(secondary, 400, 20, 24),
      margin-bottom: 8px !important;
      max-width: 100% !important;
      order: 2 !important;
```

kenapa tydac pakay pake mixin?
trus !important-nya kok banyak yak, kenapa ini?

---

ketimpa sama inline style kak, tadi aku coba jadi ga kebaca kalau pake mixin